src/public/modules/forms/services/form-api.client.factory.js

```diff
@@ -98,6 +99,7 @@ function FormApi($resource, FormErrorService, FormFields) {
         // disable IE ajax request caching (so new fields show on build panel)
       },
       getPublic: {
+        url: V3_PUBLICFORM_URL,
         method: 'GET',
         // disable IE ajax request caching (so new fields show on build panel)
         headers: { 'If-Modified-Since': '0' },
```

Comment:
you can delete the line immediately after this one, since we don't need the `accessMode` param anymore for this